### DISCUSSION TITLE
Effect of C rates on Degradation

### DISCUSSION THREAD

**Discussion by Ansh-Lodhi** [pybamm-team/PyBaMM#3008] [Q&A]
I was cycling DFN model with Okane parameters and both SEI and Lithium Plating turned on to check. I have the following queries.
1. Going by theory, Higher C rates should have higher degradation rates. However I observed reverse trend for partially reversible and irreversible lithium plating. I didn't check for reversible. Can somebody confirm the observation or am I doing something wrong.
![output2](https://github.com/pybamm-team/PyBaMM/assets/135127806/fc09ee85-27f4-4987-a392-fe0e7d4a1ff1)
![output1](https://github.com/pybamm-team/PyBaMM/assets/135127806/64c95ed0-fe44-4b5d-b04e-0dac57d68320)

2. In graph 1, the degradation in 500 cycles is not comparable (are much slower) to what one would expect in real cell. Can somebody confirm this as well?

Thank You

**Comment by brosaplanella** [comment]
1. This effect (1C ageing faster than 2C) is something that is also observed in the experiments. I am not sure what causes it, but we consistently see it in both models and experimental data.
2. This is probably caused by the parameter values. Note that parameterising degradation is still an ongoing research effort and the parameters we have in PyBaMM are more as an illustrative example. If you need to predict a specific battery you would need to get the parameters first.

**Reply by Ansh-Lodhi** [reply]
Could you provide reference for 1. From what I learnt, lithium plating is dominant at low temperature or higher C rates and I was expecting the same. Any reference would be helpful.
Thank You

**Reply by brosaplanella** [reply]
Unfortunately I don't have a reference. This is something I have seen in some data from collaborators but there is no publication yet.

**Reply by TomTranter** [reply]
How do the graphs look when plotted against time rather than cycle number?

**Reply by rtimms** [reply]
^this. for a fixed number of cycles, using a lower rate results in a longer calendar time. not sure exactly which models/parameters you’re using, but you might expect some [calendar aging](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/scripts/calendar_ageing.py)

**Reply by Ansh-Lodhi** [reply]
The trend is right when compared with respect to time, i.e. more degradation at higher C rates.

**Reply by Ansh-Lodhi** [reply]
I am still unable to get convincing explaination about the trends. Even though the amount of time spent by 2C cell in operation is much less than that spent by 1C, the above graph would implicate that we would be obtain more number of cycles from 2C cell, which doesn't make sense to me. I have tried manipulation degradation parameters to get correct trends but haven't been successful.

**Reply by TomTranter** [reply]
maybe @DrSOKane has some suggestions

**Comment by Ansh-Lodhi** [comment]
Observed Similar Trend for DFN model with just SEI turned on : 
![output](https://github.com/pybamm-team/PyBaMM/assets/135127806/6e596ffe-1b91-4138-b9db-d07997d68f99)

**Reply by ben-kenney** [reply]
I just found out about pybamm a few days ago (what a great tool by the way!) and so I can't really contribute too much but I have seen this behaviour before in lithium ion battery simulations. At the time I rationalized it as being time based as well. In very simple terms capacity fade is related to the integral of the side reation current density over time. At 2C this reaction takes place over a shorter period of time than at 1C...so as others have said, time based.

I'm also curious: do you have the energy solver turned on? The other reason why high C-rate is harmful is because of the increase in temperature since the side reactions follow Arrhenius relationship.

**Reply by Ansh-Lodhi** [reply]
As far as I know, the degradation models used in PyBaMM are not temperature dependent.

**Reply by ben-kenney** [reply]
If you don't have temperature dependent parameters you may need to create your own parameter set by fitting against test data.

There's some [relevant discussion here](https://github.com/pybamm-team/PyBaMM/discussions/2992) regarding SEI layer growth but IMHO finding the correct parameter set for capacity fade is as much art as it is science.

**Reply by Ansh-Lodhi** [reply]
That indeed might work out, will share the results if I am able to get right curves

**Comment by DrSOKane** [comment]
Hi everyone, sorry for the late response but it is exam season!

The only mechanisms that will always get worse with increasing current are the mechanical ones i.e. particle cracking and stress-driven loss of active material. SEI depends entirely on what option you chose. For the "solvent-diffusion limited" option I used for my paper, there is no link between SEI and C-rate at all unless the "SEI on cracks" option is enabled. Plating depends on both current and time. Slow charging at high states of charge is worse than fast charging at low states of charge.
